server/lib/routes/get-index.js

```diff
@@ -3,13 +3,18 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var flowMetrics = require('../flow-metrics');
+var uaParser = require('node-uap');
 
 module.exports = function (config) {
   var ALLOWED_PARENT_ORIGINS = config.get('allowed_parent_origins');
   var AUTH_SERVER_URL = config.get('fxaccount_url');
   var CLIENT_ID = config.get('oauth_client_id');
   var ENV = config.get('env');
   var FLOW_ID_KEY = config.get('flow_id_key');
+  var FIREFOX_IOS_DOWNLOAD_LINK = 'https://app.adjust.com/2uo1qc?' +
+    'campaign=fxa-devices-page&adgroup=ios&creative=button' +
+    '&fallback=https://itunes.apple.com/app/apple-store/id989804926?pt=373246&ct=adjust_tracker&mt=8';
+  var IOS_BANNER_META = '<meta name="apple-itunes-app" content="app-id=989804926, app-argument=' + FIREFOX_IOS_DOWNLOAD_LINK + '">';
```

Comment:
@vbudhram might need to add a flag here for itunes / app store to signal that this is coming from fxa